### PR TITLE
feat(tokens): Add/update theme tokens

### LINKED ISSUE #26794: [Feature]: Neutral Stroke 4 token
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

Requesting one new token
colorNeutralStrokeSubtle

Screenshot 2023-03-03 at 2 21 01 PM
https://www.figma.com/file/KB9oUjMKen2cKnyPG7RgdS/Design-tokens-superset?node-id=38002%3A79865&t=Mupzpdnt6AqP3byx-4
### Have you discussed this feature with our team

Yes

### Additional context

For use of dividers for headers and panels. needed a darker color for dark theme
image
image
image
image

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### LINKED ISSUE #27055: [Feature]: Update to brand tint token Brand Foreground 2 and Brand Background 2 to include interactive states.
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

Based on several product team requests and Fluents upcoming branded pill, we need to 

(1) make changes to Brand Background 2 dark theme values and,
(2) Add four new alias tokens to support interactive states.

### Fluent 2 tokens;
CleanShot 2023-03-03 at 15 37 52@2x

- [ ] Change Brand Background 2 Dark: Comm 20
- [ ] Add Brand Foreground 2 Hover Light: Comm 60, Dark: Comm 130
- [ ] Add Brand Foreground 2 Pressed Light: Comm 30, Dark: Comm 160
- [ ] Add Brand Background 2 Hover Light: Comm 150, Dark: Comm 40
- [ ] Add Brand Background 2 Pressed Light: Comm 130, Dark: Comm 10

### Teams tokens;
CleanShot 2023-03-03 at 15 37 33@2x

- [ ] Change Brand Background 2 Dark: Teams 20
- [ ] Add Brand Foreground 2 Hover Light: Teams 60, Dark: Teams 130, HC: buttonFace
- [ ] Add Brand Foreground 2 Pressed Light: Teams 30, Dark: Teams 160, HC: buttonFace
- [ ] Add Brand Background 2 Hover Light: Teams 150, Dark: Teams 40, HC: Highlight
- [ ] Add Brand Background 2 Pressed Light: Teams130, Dark: Teams 10, HC: Highlight

### Have you discussed this feature with our team

Yes

### Additional context

https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/concepts-introduction--page

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### LINKED ISSUE #27704: [Feature]: Add interactive brand tint stroke tokens. Brand Stroke 2 Hover / Pressed
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

In order to support our copilot prompt suggestions (and the new [interactive tint colors](https://github.com/microsoft/fluentui/issues/27055)) we identified a need for interactive brand tint stroke colors.

Add the following tokens;

- Brand Stroke 2 Hover
- Brand Stroke 2 Pressed

### Fluent
CleanShot 2023-04-26 at 13 27 19@2x

### Teams
CleanShot 2023-04-26 at 13 31 08@2x

For more context, [here is a link to the design token request](https://www.figma.com/file/HWqlLLIyn0qsRBQ8FDk58t/Design-token-requests?node-id=1314%3A140589&t=EellgwPkKN6T2H3R-1).

### Have you discussed this feature with our team

Yes.

### Additional context

_No response_

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by miroslavstastny**
### Updated tokens

| token | Light | Dark | Teams HC | Teams Dark | GH issue |
|---|---|---|---|---|---|
| `BrandForeground2` before | brand[70] | brand[110] | hcButtonText | brand[120] | #27055 |
| `BrandForeground2` after  | brand[70] | brand[110] | **hcCanvasText** | brand[120] | #27055 |
| `BrandBackground2` before | brand[160] | brand[40] | hcButtonFace | brand[40] | #27055 |
| `BrandBackground2` after  | brand[160] | **brand[20]** | **hcCanvas** | **brand[20]** | #27055 |

#### Before
image

#### After
image

### Added tokens

| token | Light | Dark | Teams HC | Teams Dark | GH issue |
|---|---|---|---|---|---|
| BrandForeground2Hover | brand[60] | brand[130] | hcCanvas | brand[130] | #27055 |
| BrandForeground2Pressed | brand[30] | brand[160] | hcCanvas | brand[160] | #27055 |
| BrandBackground2Hover | brand[150] | brand[40] | hcCanvas | brand[40] | #27055 |
| BrandBackground2Pressed | brand[130] | brand[10] | hcCanvas | brand[10] | #27055 |
| NeutralStrokeSubtle | grey[88] | grey[4] | hcCanvasText | grey[4] | #26794 |
| BrandStroke2Hover | brand[120] | brand[50] | hcHighlight | brand[50] | #27704 |
| BrandStroke2Pressed | brand[80] | brand[30] | hcHighlight | brand[30] | #27704 |

#### After
image

## Related Issue(s)

- Fixes #27055
- Fixes #26794
- Fixes #27704

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                              |    Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                  Change |
| :------------------------------------------------------------------------------------------------------------- | --------------------------: | --------------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>   |  `31.33 kB`<br />`6.764 kB` |  `31.73 kB`<br />`6.813 kB` | `400 B` <br />`49 B`  |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr> | `17.895 kB`<br />`5.162 kB` | `18.033 kB`<br />`5.181 kB` | `138 B` <br />`19 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                   |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr> | `86.012 kB`<br />`26.086 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                             |  `23.512 kB`<br />`7.197 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                               |  `24.416 kB`<br />`7.506 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                             |  `31.904 kB`<br />`8.296 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                          |  `36.724 kB`<br />`9.458 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                          | `43.873 kB`<br />`10.939 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                  | `41.411 kB`<br />`10.791 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                | `49.635 kB`<br />`12.364 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                              | `54.814 kB`<br />`11.358 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                       |  `17.399 kB`<br />`6.298 kB` |
| <samp>react-field</samp> <br /> <abbr title='bundle-size/Field.fixture.js'>Field</abbr>                                             |  `18.864 kB`<br />`7.004 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                             |  `11.479 kB`<br />`4.573 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                             |  `10.104 kB`<br />`4.185 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                               |  `12.304 kB`<br />`5.061 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                     |  `11.214 kB`<br />`4.271 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                          |  `11.649 kB`<br />`4.262 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                          |  `24.008 kB`<br />`8.798 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                        |  `18.033 kB`<br />`6.666 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                       |  `20.882 kB`<br />`6.798 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                              |   `12.492 kB`<br />`4.92 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                            |  `15.624 kB`<br />`5.232 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>              |           `69 B`<br />`89 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659'>3aecae7fc8af9dc0c4de35f58a61b76ab9e9a659</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount_base.html">118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount.html">128</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount_base.html">19</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Avatar-mount_base.html">692</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Avatar-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Button-mount_base.html">399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Button-mount.html">381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Field-mount_base.html">1285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Field-mount.html">1271</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProvider-mount_base.html">933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProvider-mount.html">925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount_base.html">118</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount.html">128</a></td>
            <td>10</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">96</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">109</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">108</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">108</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount_base.html">19</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount.html">21</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/InfoButton-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/MakeStyles-mount_base.html">1143</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/MakeStyles-mount.html">1149</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Persona-mount_base.html">2082</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/Persona-mount.html">1967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/SpinButton-mount_base.html">1560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27791/perf-test-react-components/SpinButton-mount.html">1561</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts [hunk 1]
```diff
@@ -135,6 +135,8 @@ export const alias = {
   colorCompoundBrandForeground1Pressed: 'brand[60]',
```
Comment (fabricteam) [inline apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:135]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 9 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27791&commitId=7e2af887ed85dd2346db8bbdcafa33256263e3e3&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1924 screenshots unchanged, 0 screenshots with different dimensions and 9 screenshots with visible difference.


  <details><summary><b>unknown</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Badge Converged.tint - Dark Mode.chromium.png|217| Changed|
|Badge Converged.tint - High Contrast.chromium.png|393| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|12| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|24857| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|12405| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|6200| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|24857| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|12405| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|6200| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Comment (miroslavstastny) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:135]: @redongreen or @daisygeng:
Needs design review for `Table` and `Badge`.
Breaks `Badge` and `DatePicker` in Teams HC theme.

File: packages/tokens/src/alias/teamsDarkColor.ts [hunk 2]
```diff
@@ -38,7 +38,9 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorCompoundBrandForeground1Hover: brand[110], // #3aa0f3 Global.Color.Brand.110
   colorCompoundBrandForeground1Pressed: brand[90], // #1890f1 Global.Color.Brand.90
   colorBrandForeground1: brand[100], // #2899f5 Global.Color.Brand.100
-  colorBrandForeground2: brand[120], // #6cb8f6 Global.Color.Brand.120
+  colorBrandForeground2: brand[110], // #3aa0f3 Global.Color.Brand.110
```
Comment (miroslavstastny) [inline packages/tokens/src/alias/teamsDarkColor.ts:41]: @redongreen - `colorBrandForeground2` in Teams Dark theme was previously changed from Brand.110 to Brand.120 to fix #23898. Are we sure we want to assign this back to 110?

File: packages/tokens/src/alias/highContrastColor.ts [hunk 3]
```diff
@@ -48,7 +48,9 @@ export const generateColorTokens = (): ColorTokens => ({
   colorCompoundBrandForeground1Hover: hcHighlight, // Highlight Global.Color.hcHighlight
   colorCompoundBrandForeground1Pressed: hcHighlight, // Highlight Global.Color.hcHighlight
   colorBrandForeground1: hcCanvasText, // CanvasText Global.Color.hcCanvasText
-  colorBrandForeground2: hcButtonText, // ButtonText Global.Color.hcButtonText
+  colorBrandForeground2: hcCanvasText, // CanvasText Global.Color.hcCanvasText
```
Comment (miroslavstastny) [inline packages/tokens/src/alias/highContrastColor.ts:51]: The change of `colorBrandForeground2` and `colorBrandBackground2` effectively means switching the colors from black-on-white to white-on-black. This change has already been incorporated on partner side by using an override.

But it breaks `Badge` and `DatePicker`, see screenshot test comment.

File: packages/tokens/src/alias/teamsDarkColor.ts (hunk 2 above)
Comment (miroslavstastny) [reply packages/tokens/src/alias/teamsDarkColor.ts]: After discussion with @daisygeng reverting back to `brand[120]` in Teams Dark.

File: apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts (hunk 1 above)
Comment (miroslavstastny) [reply apps/public-docsite-v9/src/Concepts/Migration/FromV8/ThemeColors/colorMap.ts:135]: The `DatePicker` change is actually a fix.
`Table` changes handled by @ling1726.
`Badge` changes are expected.

**Review by ling1726 — APPROVED** [review]

**Review by GeoffCoxMSFT — APPROVED** [review]

**Pull request merged**